get-training-set.sh

```
@@ -0,0 +1,216 @@
+    BASE_URL="${RETROCAST_TRAINING_SET_BASE_URL:-https://files.ischemist.com/retrocast/training-sets}"
+    CACHE_DIR="${RETROCAST_TRAINING_SET_CACHE_DIR:-${XDG_CACHE_HOME:-$HOME/.cache}/retrocast/training-sets}"
+    OUTPUT_DIR=""
+    DATASET="paroutes"
+    ARTIFACT=""
+    SPLIT="training"
+    FORMAT="auto"
+    RELEASE="latest"
+    DRY_RUN=0
+
+    R='\033[0;31m'
+    G='\033[0;32m'
```

Shellcheck (0.11.0)

[warning] G appears unused. Verify use (or export if used externally).

(SC2034)